src/partials/3-boxes/style.scss

```diff
@@ -24,7 +24,7 @@ $yellow-dark: darken($youth-yellow, 15%);
 
   .btn {
     padding: 0.75rem 4rem;
-    background: $youth-yellow;
+    background: $flame-yellow;
     box-shadow: 0 2px 0 $yellow-dark;
```

Comment:
Yeah, that's why I changed it to flame, because youth was missing. But since we're just using it once I can hard code the darkened version...